receiver/mysqlreceiver/testdata/dockerfiles/Dockerfile.mysql.8_0

```diff
@@ -0,0 +1,13 @@
+FROM mysql:8.0.25
```

Comment:
Can we rename this directory to `mysqlreceiver/testdata/integration`?

Reply:
renamed